modules/internal-load-balancer/main.tf

```diff
@@ -1,165 +1,96 @@
 locals {
-  name_in  = "${local.prefix}in"
-  name_out = "${local.prefix}out"
-  ports    = [var.vpc_cluster_assistant_tcp_port, var.vpc_kubernetes_tcp_port]
-  prefix   = "${var.prefix}ilb-"
+  application_name = "${var.prefix}private-application"
 }
 
-resource "google_compute_health_check" "internal_load_balancer_out" {
-  name = local.name_out
+resource "google_compute_region_health_check" "application" {
+  provider = google-beta
 
-  check_interval_sec = 10
-  description        = "Check the health of the Kubernetes API."
-  tcp_health_check {
-    port = var.vpc_kubernetes_tcp_port
+  name = local.application_name
+
+  check_interval_sec = 5
+  description        = "TFE application."
+  https_health_check {
+    port         = var.vpc_application_tcp_port
+    request_path = "/_health_check"
+  }
+  # Beta
+  log_config {
+    enable = true
   }
-  unhealthy_threshold = 3
+  timeout_sec = 4
```

Comment:
is there any tuning we have done to get this number, or is it just a reasonable starting default?

Reply:
looks like this was taken from the timeout used previously.

Reply:
Yeah, I'm not aware of any basis for deviations from default values in regards to timeouts, rate limits, and the like. We can drop them all until we have a clear justification to change them. What do you think?

Reply:
I think they're fine as is for now, but I also feel that we should keep an eye on making a project around tuning these and making sure they're not adding brittleness to the system.